addons/mail/models/ir_cron.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from odoo import api, fields, models
+
+
+class ir_cron(models.Model):
+    _inherit = 'ir.cron'
+
+    @api.model
+    def _handle_callback_exception(self, cron_name, server_action_id, job_id, job_exception):
+        """
+            Method that overrides the original one from base to post a notification on admins channel.
+        """
+        super(ir_cron, self)._handle_callback_exception(cron_name, server_action_id, job_id, job_exception)
+        # Get datetime when error occured != self.search([('id', '=', job_id)], limit=1).nextcall
+        cron_error_datetime = fields.Datetime.to_string(datetime.now())
+        job_record = self.search([('id', '=', job_id)], limit=1)
+        # Send notification to admin : errors related to cron job callback
+        self._notify_admins(*self._get_admin_notification('base__cron')(job_record._format_record_link(), job_exception, cron_error_datetime, self._cr.dbname))
```

Comment:
Please use the same notation that you did at : 
https://github.com/odoo/odoo/pull/50871/files#diff-5987ed2f79959b9ae0b58a54080fd214R143

Much more readable ;)